### PR TITLE
Add torchdynamo.graph_break()

### PR DESCRIPTION
**Pull request by jansel**
This is a way to force a graph break rather than relying on stuff like print.

### CODE REVIEW

**Review by voznesenskym — APPROVED** [review]

File: torchdynamo/allowed_functions.py [hunk 1]
```diff
@@ -82,15 +82,13 @@ def _disallowed_function_ids():
...
         torch.distributions.normal.Normal,
-        torch.get_rng_state,
         torch.inference_mode,
         torch.set_anomaly_enabled,
         torch.set_autocast_cache_enabled,
         torch.set_autocast_cpu_dtype,
         torch.set_autocast_cpu_enabled,
         torch.set_autocast_enabled,
         torch.set_autocast_gpu_dtype,
-        torch.set_rng_state,
```
Comment (voznesenskym) [inline torchdynamo/allowed_functions.py:93]: Why remove these?

Comment (jansel) [reply torchdynamo/allowed_functions.py:93]: Before this PR these were getting inlined, which seems incorrect.   The change below makes them not inline anymore.

Comment (voznesenskym) [reply torchdynamo/allowed_functions.py:93]: Nice :)

**Comment by jansel** [comment]
#1615

**Pull request closed without merging**